### ISSUE TITLE
NoClassDefFoundError on eclipse 4.23 on mac

### ISSUE THREAD

**Issue by alexgordtop** [JorisAerts/Eclipse-Environment-Variables#17]
```
java.lang.NoClassDefFoundError: Could not initialize class com.jorisaerts.eclipse.rcp.environment.environment.internal.Environment
	at com.jorisaerts.eclipse.rcp.environment.environment.EnvironmentVariables.remove(EnvironmentVariables.java:29)
	at com.jorisaerts.eclipse.rcp.environment.util.EnvironmentVariablesUtil.clearOldVariables(EnvironmentVariablesUtil.java:25)
	at com.jorisaerts.eclipse.rcp.environment.util.EnvironmentVariablesUtil.setEnvironmentVariables(EnvironmentVariablesUtil.java:41)
	at com.jorisaerts.eclipse.rcp.environment.preferences.EnvironmentPreferencePage.performOk(EnvironmentPreferencePage.java:80)
...
```

eclipse.buildId=4.23.0.I20220308-0310
java.version=17.0.2
java.vendor=Eclipse Adoptium
BootLoader constants: OS=macosx, ARCH=aarch64, WS=cocoa, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.java.product -product org.eclipse.epp.package.java.product -product org.eclipse.epp.package.java.product -keyring /Users/www/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch aarch64 -product org.eclipse.epp.package.java.product -product org.eclipse.epp.package.java.product -data file:/Users/www/WS/2022-03-java/ -product org.eclipse.epp.package.java.product -keyring /Users/www/.eclipse_keyring

**Comment by bachden** [comment]
same here.

**Comment by JorisAerts** [comment]
fixed in v1.0.0
